main.py

```diff
@@ -711,6 +717,58 @@ def make_positive_or_negative(self) -> None:
 
             self._positive_or_negative_json["data"].append(data)
 
+    def make_warning_and_phase(self) -> None:
+        # テンプレートから変数を初期化
+        self._warning_and_phase_json = {
+            "data": {},
+            "last_update": self.get_inspections_last_update()
+        }
+
+        warning = 0
+        phase = 0
+
+        try:
+            positive_or_negative_dict = self.positive_or_negative_json()
+            average_length = len(positive_or_negative_dict['data'])
+            latest_average_patients = positive_or_negative_dict['data'][average_length - 1]['7日間平均陽性数']
+        except Exception:
+            raise Exception("Failed get positive or negative data.")
+
+        try:
+            soup = get_html_soup(base_url)
+        except Exception:
+            raise Exception(f"Failed get \"{base_url}\".")
```

Comment:
```suggestion
        soup = get_html_soup(base_url)
```
util.pyの`get_file`関数内では別にエラーハンドリングしてないのでこちらも別に良いかなと